pkg/storaged/stratis-details.jsx

```diff
@@ -398,6 +547,56 @@ export const StratisPoolDetails = ({ client, pool }) => {
                         </DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    { pool.Encrypted && client.features.stratis_crypto_binding &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm className="control-DescriptionListTerm">
+                            {_("storage", "Passphrase")}
+                        </DescriptionListTerm>
+                        <DescriptionListDescription>
+                            { key_desc ? cockpit.format(_("using key description \"$0\""), key_desc) : <i>{_("not configured")}</i> }
+                            <DescriptionListDescription className="tab-row-actions">
+                                { !key_desc
+                                    ? <StorageButton onClick={add_passphrase}>{_("Add")}</StorageButton>
+                                    : null
+                                }
+                                { key_desc
+                                    ? <>
```

Comment:
Curious that eslint allows that -- it seems it always yells at me when opening and closing tag are not aligned. :man_shrugging: if it passes, fine for me :grin:

Reply:
Yes, this is the result of eslint:fix, I am pretty sure.